Foxtrot/Driver/Program.cs

```diff
@@ -1333,25 +1333,25 @@ private static int PEVerify(string assemblyFile)
             }
 
             // Extract the contracts from the code (includes checking the contracts)
+            
+            string contractFileName = Path.GetFileNameWithoutExtension(assemblyNode.Location) + ".Contracts";
 
-            if (!options.passthrough)
-            {
-                string contractFileName = Path.GetFileNameWithoutExtension(assemblyNode.Location) + ".Contracts";
-
-                if (options.contracts == null || options.contracts.Count <= 0) contractFileName = null;
+            if (options.contracts == null || options.contracts.Count <= 0) contractFileName = null;
 
-                if (options.contracts != null &&
-                    !options.contracts.Exists(
-                        name => name.Equals(assemblyNode.Name + ".Contracts.dll", StringComparison.OrdinalIgnoreCase)))
-                    contractFileName = null;
+            if (options.contracts != null &&
+                !options.contracts.Exists(
+                    name => name.Equals(assemblyNode.Name + ".Contracts.dll", StringComparison.OrdinalIgnoreCase)))
```

Comment:
I known this is an old code, but for such a complex `if` statement that we're touching anyway, I would suggest to add curly braces..